.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: rust
+
+rust:
+ - 1.31.0
```

Comment:
1.31.0 was picked as it's the minimum needed for Rust 2018, and that seems to be enough